event-tool/src/test/java/uk/gov/justice/event/tool/StartTransformationTest.java

```diff
@@ -158,19 +168,28 @@ public void shouldRemoveFinishedTaskForAllPasses() {
 
         startTransformation.taskDone(future3, null, null, null);
         assertThat(startTransformation.outstandingTasks.size(), is(0));
+
+//        verify(linkedEventStreamTransformationService).truncateLinkedEvents();
+//        verify(linkedEventStreamTransformationService).populateLinkedEvents();
     }
 
 
     @Test(expected = IllegalArgumentException.class)
     public void shouldThrowExceptionWhenstreamsProcessedCountStepInfoIsNull() throws Exception {
         streamsProcessedCountStepInfo.set(startTransformation, null);
         startTransformation.go();
+//
+//        verify(linkedEventStreamTransformationService).truncateLinkedEvents();
+//        verify(linkedEventStreamTransformationService).populateLinkedEvents();
```

Comment:
this comment code should be reinstated or deleted

Reply:
done as mentioned by allan and martin